test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestAttachAPIEnabling.java

```diff
@@ -30,17 +30,14 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import com.ibm.tools.attach.target.AttachHandler;
-
 /**
  * This test must be invoked via testng.  Running the main method will return only the status of the attach API.
  *
  */
-@SuppressWarnings({"nls","boxing"})
+@SuppressWarnings({"nls"})
 public class TestAttachAPIEnabling extends AttachApiTest {
-	private static final Integer ATTACH_API_ENABLED_CODE = 12;
-	private static final Integer ATTACH_API_DISABLED_CODE = 13;
-	private static boolean verbose = false;
+	private static final Integer ATTACH_API_ENABLED_CODE = Integer.valueOf(12);
+	private static final Integer ATTACH_API_DISABLED_CODE = Integer.valueOf(13);
```

Comment:
Why doesn't this use `int` rather than `Integer`? Using primitive types should be preferred over box types.

Reply:
>  Why doesn't this use `int` rather than `Integer`? 

No good reason.  I'll be glad to clean it up.